patterns/goodreads-signin.html

```diff
@@ -3,8 +3,7 @@
     <title>Goodreads Sign In</title>
   </head>
   <body>
-    <h4 gg-optional gg-match=".a-alert-heading"></h4>
-    <p gg-optional gg-match=".a-alert-content"></p>
+    <p gg-optional gg-match="div[aria-live] .a-alert-content"></p>
```

Comment:
just wonder why some using XPath and the other using css selector?

Reply:
In the above description, I wrote the lack of support for some pseudo-classes in Zendriver, e.g. `has-text()`. To compensate that, we should fall back to XPath.

Technically nothing stops us from using XPath everywhere. However XPath selectors are cumbersome for simple cases, hence it's better to use CSS selector by default, unless `has-text` or `contains` is needed.